source/content/relaunch.md

```diff
@@ -77,6 +78,8 @@ For a fast, smooth relaunch, consider having two browser tabs open, one with the
 
   Do **not** update DNS yet. The Site Dashboard instructs you to configure DNS after connecting the domain, however for this relaunch procedure you should not change record values until instructed in step 5 below.
 
+  Until you update the DNS records (and they propogate), traffic will still be directed to the old site.
```

Comment:
This is the only line related to the issue, the rest is me scope-creeping like a mutha.